cli/src/main.rs

```diff
@@ -40,6 +41,14 @@ struct Opt {
     /// Paths to search
     #[structopt(value_name = "path")]
     paths: Vec<String>,
+    /// Paths to exclude
+    ///
+    /// For example:
+    /// --exclude-path=005_user_ids.sql --exclude-path=009_account_emails.sql
+    ///
+    /// --exclude-path='*user_ids.sql'
+    #[structopt(long = "exclude-path")]
+    excluded_path: Option<Vec<String>>,
```

Comment:
I removed the use_delimiter setting because I think paths can containe `,`. Now users can specify the `--exclude-path` flag multiple times instead.